configs/default/EACH-EACHINEF722.config

```diff
@@ -0,0 +1,132 @@
+# Betaflight / STM32F405 (S405) 4.2.0 Dec  4 2019 / 01:50:58 (ebdda3dec7) MSP API: 1.43
```

Comment:
Please test your targets with released firmware and not with development builds.